TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/SimpleFileBackend.php

```diff
@@ -257,14 +252,7 @@ public function remove($entryIdentifier)
 
         $pathAndFilename = $this->cacheDirectory . $entryIdentifier . $this->cacheEntryFileExtension;
 
-        try {
-            $lock = new Lock($pathAndFilename);
-            unlink($pathAndFilename);
-            $lock->release();
-        } catch (\Exception $exception) {
-            return false;
-        }
-        return true;
+        return @unlink($pathAndFilename);
```

Comment:
Hm, yeah I see... basically means that on Windows the delete will just fail if it is concurrently read somewhere else and on *NIX I'm not sure if the read would just create currupt data, fail or if the delete would fail. Do you have any idea what the behaviour is?

What if we keep the lock on non-windows only? That way we retain the concurrent-read safety on like 99% of used production environments and at least have an as-good-as-possible behaviour no windows.